src/components/sectionsWheelPicker/SectionsWheelPicker.driver.tsx

```diff
@@ -4,8 +4,8 @@ import {WheelPickerDriver} from '../WheelPicker/WheelPicker.driver';
 import {SectionsWheelPickerProps} from './index';
 
 export const SectionsWheelPickerDriver = (props: ComponentProps) => {
-  const driver = useComponentDriver<SectionsWheelPickerProps>(props);
-  const sections = driver.getProps().children as SectionsWheelPickerProps;
+  const driver = useComponentDriver(props);
+  const sections = driver.getElement().children as SectionsWheelPickerProps;
```

Comment:
Why do we need the `SectionsWheelPickerProps`?

Reply:
This is needed so TS will enable us to get the testID from the children. This is not related to this PR though.

Reply:
I don't see a usage of `driver.sections` in private, and I don't see an error in public...

Reply:
Its used here
https://github.com/wix/react-native-ui-lib/blob/43151d5a2e129c3fc9f39bbb8c1fefbb63366536/src/components/sectionsWheelPicker/__tests__/index.spec.tsx#L33-L39

Reply:
I still don’t understand why we need the type